scripts/speech_recognition/convert_to_tarred_audio_dataset.py

```diff
@@ -81,6 +88,13 @@
 from joblib import Parallel, delayed
 from omegaconf import DictConfig, OmegaConf, open_dict
 
+try:
+    import create_dali_tarred_dataset_index as dali_index
+
+    DALI_INDEX_SCRIPT_AVAILABLE = True
```

Comment:
why would `create_dali_tarred_dataset_index` be not available? It is part of this repro. Shouldn't  `dali_index.INDEX_CREATOR_AVAILABLE` be enough?

Reply:
Dali is an optional install that the user has to actively do, whereas the tarred dataset building script can be used even without even installing Nemo itself. So we test and then build if Dali is available (even if Nemo is not)